src/enumerable.cr

```diff
@@ -1241,6 +1241,17 @@ module Enumerable(T)
     count { true }
   end
 
+  # Returns `true` if `self` is empty, `false` otherwise.
+  #
+  # ```
+  # ([] of Int32).empty? # => true
+  # ([1]).empty?         # => false
+  # ```
+  def empty?
+    each { return true }
+    false
+  end
```

Comment:
This is backwards:

https://carc.in/#/r/8smy

Reply:
lol 🤦‍♂, good catch!